testbed/tests/trace_test.go

```diff
@@ -53,26 +53,26 @@ func TestTrace10kSPS(t *testing.T) {
 			testbed.NewJaegerGRPCDataSender(testbed.GetAvailablePort(t)),
 			testbed.NewJaegerDataReceiver(testbed.GetAvailablePort(t)),
 			testbed.ResourceSpec{
-				ExpectedMaxCPU: 69,
-				ExpectedMaxRAM: 89,
+				ExpectedMaxCPU: 80,
+				ExpectedMaxRAM: 50,
 			},
 		},
 		{
 			"OpenCensus",
 			testbed.NewOCTraceDataSender(testbed.GetAvailablePort(t)),
 			testbed.NewOCDataReceiver(testbed.GetAvailablePort(t)),
 			testbed.ResourceSpec{
-				ExpectedMaxCPU: 42,
-				ExpectedMaxRAM: 84,
+				ExpectedMaxCPU: 52,
+				ExpectedMaxRAM: 46,
 			},
 		},
 		{
 			"OTLP",
 			testbed.NewOTLPTraceDataSender(testbed.GetAvailablePort(t)),
 			testbed.NewOTLPDataReceiver(testbed.GetAvailablePort(t)),
 			testbed.ResourceSpec{
-				ExpectedMaxCPU: 60,
-				ExpectedMaxRAM: 84,
+				ExpectedMaxCPU: 80,
```

Comment:
why 80?

Reply:
Without batch processor it consumes more CPU and less memory

Reply:
makes sense :)